.github/workflows/profiling.yaml

```diff
@@ -40,78 +27,58 @@ jobs:
           echo "Context: "
           cat ${GITHUB_EVENT_PATH}
 
-          echo "Creating Download URL from ISSUE"
-          if ! echo "DOWNLOAD_URL=$(cat ${GITHUB_EVENT_PATH} | jq '.comment.body' | grep -oP '(https:)(.*?)(?=\))' )" >> ${GITHUB_ENV}; then
```

Comment:
Reading through this, I thought we might want to check whether the DOWNLOAD_URL is going to work already here... 
could use `curl --head --fail $DOWNLOAD_URL` to test it without an actual download.